Source/ACE.Entity/Enum/Active.cs

```diff
@@ -0,0 +1,9 @@
+namespace ACE.Entity.Enum
```

Comment:
this is unneeded, as there is currently no defined logic based on the difference between 'Active' and 'SecondaryAction'